File: Challenge1/fizzbuzz.py
print '\n\n\nChallenge 1\n'
def fizz_buzz(n):
	result = ''
	if not n % 3:
		result += 'Fizz'
	if not n % 5:
		result += 'Buzz'
	if n % 3 and n % 5:
		result = n
	return result

def fizz_buzz_to_n(n):
	for i in xrange(n):
		print fizz_buzz(i)

fizz_buzz_to_n(100)

File: Challenge1/fizzbuzz.rb
args = []

ARGV.each_with_index do|a,i|
  args[i] = a
end

def fizzbuzz(number)
    divisibleBy3 = (number % 3 == 0)
    divisibleBy5 = (number % 5 == 0)

    case
        when divisibleBy3 && divisibleBy5
            puts "FizzBuzz"
        when divisibleBy3
            puts "Fizz"
        when divisibleBy5
            puts "Buzz"
        else
            puts number
    end
end

startNbr = args[0].to_i
endNbr   = args[1].to_i

if (startNbr <= endNbr)
    (startNbr..endNbr).each {|n| fizzbuzz n}
else
    puts "First arg must be less than or equal to second arg."
end
